tests/test_model_functions.py

```diff
@@ -84,6 +84,51 @@ def test_get_derived_quantities(self, basic_model, simple_ode_model):
             '_deriv_on_rhs2b$sv1_rate]'
         )
 
+    def test_get_display_name(self, simple_ode_model):
+        """ Tests Model.get_display_name(var). """
+        # For getting a display name there are 4 possibilities:
+        # - Has an oxmeta-annotation
+        # - Has an annotation in another ontology
+        # - Has no annotation but does have a cmeta:id
+        # - Has no cmeta:id
+
+        # Check our 4 example variables exist
```

Comment:
Rather than this long-winded approach, can't you just do e.g.
```python
    var_oxmeta = simple_ode_model.get_variable_by_name('single_independent_ode$sv1')
    assert var.cmeta_id == 'sv11'
    assert simple_ode_model.get_display_name(var, ontology=OXMETA) == 'sodium_reversal_potential'
```